packages/manager/apps/key-management-service/src/components/layout-helpers/Dashboard/GeneralInformationsTiles/InformationsTile.tsx

```diff
@@ -86,7 +87,19 @@ const InformationsTile = ({ okmsData }: InformationTileProps) => {
           >
             {t('key_management_service_dashboard_field_label_id')}
           </OsdsText>
-          <OsdsClipboard className="mb-2" value={okmsData?.id}></OsdsClipboard>
+          <OsdsClipboard className="mb-2" value={okmsData?.id}>
+            <span slot="success-message">
+              <OsdsText color={ODS_THEME_COLOR_INTENT.success}>
+                {tCommon('clipboard_copy_success')}
+              </OsdsText>
+            </span>
+            <span slot="error-message">
+              {' '}
+              <OsdsText color={ODS_THEME_COLOR_INTENT.error}>
+                {tCommon('clipboard_copy_error')}
+              </OsdsText>
+            </span>
+          </OsdsClipboard>
```

Comment:
Without necessary create a new component you can, at least, create a function to avoid duplicated the code in the component 
```suggestion
const createOsdsClipboard = (value : any) => {
          <OsdsClipboard className="mb-2" value={value}>
            <span slot="success-message">
              <OsdsText color={ODS_THEME_COLOR_INTENT.success}>
                {tCommon('clipboard_copy_success')}
              </OsdsText>
            </span>
            <span slot="error-message">
              {' '}
              <OsdsText color={ODS_THEME_COLOR_INTENT.error}>
                {tCommon('clipboard_copy_error')}
              </OsdsText>
            </span>
          </OsdsClipboard>
    }
         ....
          {createOsdsClipboard(okmsData?.id)}
```